src/visualization/components/internal/StaticLegend.tsx

```diff
@@ -81,7 +82,7 @@ const StaticLegend: FC<Props> = ({properties, update}) => {
   const {valueAxis} = staticLegend
   const {
     colorizeRows = false, // undefined is considered false because of omitempty
-    heightRatio: defaultHeightRatio = STATIC_LEGEND_HEIGHT_RATIO_NOT_SET,
+    heightRatio: defaultHeightRatio = STATIC_LEGEND_HEIGHT_RATIO_DEFAULT,
```

Comment:
This does not make a difference for most of the charts, since it had a default of NOT_SET (a.k.a. 0 height) hardcoded.